tests/ignite/contrib/handlers/test_clearml_logger.py

```diff
@@ -956,9 +956,13 @@ def test_distrib_nccl_gpu(distributed_context_single_node_nccl):
 @pytest.mark.skipif("NUM_TPU_WORKERS" in os.environ, reason="Skip if NUM_TPU_WORKERS is in env vars")
 @pytest.mark.skipif(not idist.has_xla_support, reason="Not on TPU device")
 def test_distrib_single_device_xla():
-    device = idist.device()
-    assert "xla" in device.type
-    _test_save_model_optimizer_lr_scheduler_with_state_dict(device)
+    with pytest.warns(
+        UserWarning,
+        match=r"`tag` parameter is mandatory and is set by default to `barrier` for xla-tpu `rendezvous` method.",
+    ):
```

Comment:
Shouldn't we fix that in our codebase instead of catching the warning in the tests and user's code ?